src/jdk.management.agent/share/classes/jdk/internal/agent/resources/agent.properties

```diff
@@ -24,7 +24,7 @@
 #
 
 agent.err.error			   = Error
-agent.err.exception                = Exception thrown by the agent 
+agent.err.exception                = Exception thrown by the agent\u0020
```

Comment:
I believe this space was just a typo and should be removed. Same for `agent.err.agentclass.failed` below and in all the other management property files.